ios/Video/Features/RCTPlayerObserver.swift

```diff
@@ -98,8 +103,14 @@ class RCTPlayerObserver: NSObject {
         }
     }
 
+    func metadataOutput(_: AVPlayerItemMetadataOutput, didOutputTimedMetadataGroups groups: [AVTimedMetadataGroup], from _: AVPlayerItemTrack?) {
+        for metadataGroup in groups {
+            _handlers?.handleTimeMetadataChange(timedMetadata: metadataGroup.items)
+        }
```

Comment:
nit: I think you should unwrap the `_handlers` first as it doesn't make sense to iterate when it's nil